DrainageSystem-1/Readme.md

```diff
@@ -55,10 +55,11 @@ Network topology using `IfcDistributionPort` and `IfcRelConnectsPorts`.
 Following files correspond to this scenario:
 
 | Filename                 | Description                                                  |
-|:------------------------ | ------------------------------------------------------------ |
+| :----------------------- | ------------------------------------------------------------ |
```

Comment:
Why this change? I don't think that this is valid markdown...

Reply:
Hmmm, I use Typora for editing markdown, and sometimes things happen. Creating a new version.

Reply:
Hm, if it's Typora it should be fine. I just haven't seen such syntax yet. May be fine.